aqua/input-leap/Portfile

```diff
@@ -52,16 +49,19 @@ if {[variant_isset debug]} {
     configure.optflags
 }
 
+variant nogui description {Don't build the GUI} {}
```

Comment:
typically we would do it the other way around: have a non-default variant to build the GUI

Reply:
I'm perfectly fine with doing it the other way around, in fact even prefer it. I'll push a new patch once done with the change.